src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserRoleService.cs

```diff
@@ -406,21 +409,26 @@ await _usersAndRolesPersistenceHelper
                 .OnSuccess(async role =>
                 {
                     return await _contentPersistenceHelper
-                        .CheckEntityExists<Release>(query => query
-                            .Include(r => r.Publication)
-                            .Where(r => r.Id == role.ReleaseId)
+                        .CheckEntityExists<UserReleaseRole>(query => query
+                            .Include(userReleaseRole => userReleaseRole.User)
+                            .Include(userReleaseRole => userReleaseRole.Release)
+                            .ThenInclude(release => release.Publication)
+                            .Where(urr => urr.ReleaseId == role.ReleaseId)
```

Comment:
This doesn't look right. This would probably get the first UserReleaseRole that matches the `releaseId` (which there could be many). Additionally, there's another `CheckEntityExists` earlier on that already makes the call for the correct UserReleaseRole, so we don't need to repeat this work.

We should move these `Includes` into the initial `CheckEntityExists` i.e.

```cs
.CheckEntityExists<UserReleaseRoles>(userReleaseRoleId, 
    query => query.Includes(...)
)
```